R/utils.R

```diff
@@ -60,6 +60,46 @@ find_phantom <- function() {
   path.expand(path)
 }
 
+phantomjs_cmd_result = function(args, wait = TRUE) {
+  # Retrieve and store output from STDOUT
+  utils::capture.output(invisible(phantom_run(args = args, wait = wait)),
+                        type = "output")
+}
+
+phantomjs_version = function() {
+  phantomjs_cmd_result("--version")
+}
+
+#' Determine if PhantomJS is Installed
+#'
+#' Verifies that a version of PhantomJS is installed and available for use
+#' on the user's computer.
+#'
+#' @return \code{TRUE} if the PhantomJS is installed. Otherwise, \code{FALSE}
+#' if PhantomJS is not installed.
+#'
+#' @export
+is_phantomjs_installed <- function() {
+  suppressMessages( !is.null(find_phantom()) )
```

Comment:
Instead of `suppressMessages`, it would be cleaner to add a `quiet` option to `find_phantom`, which defaults to `FALSE`.